tot/experiment.py

```diff
@@ -42,6 +43,7 @@ class Experiment(ABC):
     metadata: Optional[dict] = None
     save_dir: Optional[str] = None
     num_processes: int = 1
+    scaler: Scaler = None
```

Comment:
Can we introduce a `scaling_level: str ` which can be `'per_dataset'` or `'per_time_series'`. Later we can add `'per_instance'` or` 'per_batch'` if we would like to go more granular. Happy about your ideas on the exact naming.